zh-CN/mapping_dev.zh_CN

```diff
@@ -908,9 +908,9 @@ dlgSgHostingProviderEdit.tle=GitLab
 dlgSgHostingProviderSelectRepository.btn"Select"=选择
 dlgSgHostingProviderSelectRepository.col"Repository"=仓库
 dlgSgHostingProviderSelectRepository.tle=GitHub 项目
-dlgSgHttpPasswordAuthenticationFailedCause1.fur%3=The OAuth-access token could not be requested. Most likely your $1 configuration has changed and SmartGit's stored OAuth credentials are invalid.\n\nTo resolve, recreate the $2 hosting provider in the Preferences.\n\nDetails:\n\n$3
-dlgSgHttpPasswordAuthenticationFailedCause1.hdl%1=$1 OAuth-authentication failed
-dlgSgHttpPasswordAuthenticationFailedCause1.tle=HTTP authentication
+dlgSgHttpPasswordAuthenticationFailedCause1.fur%3=无法请求 OAuth 访问令牌。很可能您的 $1 配置已更改且 SmartGit 存储的 OAuth 凭据无效。\n\n要解析，请在首选项中重新创建 $2 主机提供商。\n\n详细信息：\n\n $3
+dlgSgHttpPasswordAuthenticationFailedCause1.hdl%1=$1 OAuth 身份验证失败
```

Comment:
When I translated on GitLab, OAuth was willing to reserve, and on some domestic websites, OAuth was willing to reserve, because OAuth is a standard protocol.

Reply:
You are right.